node/pkg/reporter/app.go

```diff
@@ -35,13 +32,24 @@ func (a *App) Run(ctx context.Context) error {
 	return a.startReporters(ctx)
 }
 
-func (a *App) setReporters(ctx context.Context, h host.Host, ps *pubsub.PubSub) error {
+func (a *App) setReporters(ctx context.Context) error {
 	err := a.clearReporters()
 	if err != nil {
 		log.Error().Str("Player", "Reporter").Err(err).Msg("failed to clear reporters")
 		return err
 	}
 
+	dalApiKey := os.Getenv("API_KEY")
+	if dalApiKey == "" {
+		return errorSentinel.ErrReporterDalApiKeyNotFound
+	}
+
+	chain := os.Getenv("CHAIN")
+	if chain == "" {
+		log.Warn().Str("Player", "Reporter").Msg("chain not set, defaulting to baobab")
+		chain = "baobab"
+	}
```

Comment:
but isn't the endpoint still be different for each chain, since we have separate deployments for each chain

Reply:
you were setting DalEndpoint as followed
```
fmt.Sprintf("https://dal.%s.orakl.network", chain)
```

but internal url will probably stay same for both mainnet and testnet
which will be both
`http://orakl-dal.orakl.svc.cluster.local`

Reply:
i see... i just realised this might become irrelevant after i replace rest with websocket in the next pr

Reply:
lol if you use websocket, internal implementation might change a bit. 😁

Reply:
if you think websocket implementation will take more time, I won't push you. the best would be to have everything ready tmr so that it can be deployed before wednesday. 

but if it can't be done, we can just push a deadline a bit further. I was kept seeing current reporter being unstable, and thought this renewal could make some difference and thought it'd be nice to have soon. but i won't insist we can deploy it later

Reply:
gotcha, i'll aim websocket for tomorrow 🙏